source/imaer-gml/src/main/java/nl/overheid/aerius/gml/base/source/ship/v31/GML2InlandMooring.java

```diff
@@ -95,6 +101,12 @@ private void addRoute(final IsGmlInlandShippingRoute inlandShippingRoute, final
     route.getSubSources().add(imr);
     route.setMooringAId(sourceId);
     addRouteShipsToMooringShips(inlandShippingRoute, vesselGroupEmissionValues);
+    additionalRouteStuff(inlandShippingRoute, route, source);
+  }
+
+  protected void additionalRouteStuff(final IsGmlInlandShippingRoute inlandShippingRoute, final InlandShippingEmissionSource route, final T source)
+      throws AeriusException {
+    // NO-OP for this version
```

Comment:
Can you write some additional comment on why this is a no-op. And with version I assume you refer to the IMAER version.